### PR TITLE
Add project methodology

### PR DESCRIPTION
**Pull request by cjkeilig**
* Reasons why we chose Django and not WordPress (please let me know if I am wrong on this)
* How Django and Angular fit into the goal of the project (i.e. flexibility and not reinventing the wheel)
* Useful images

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,3 +41,38 @@ advancement of students in science, technology, engineering, and mathematics.
...
+
+* Built-in user management to aid in the requirement of role-based access. 
+
+* The built-in REST framework will help the application meet the requirement of being easily accessible by external sites.
+
+  <u>Django Release History</u>
+
+![Django Release History](pics/Django_Release_History.PNG "")
+
+AngularJS
```
Comment (codykaup) [inline README.md]: This should be `Angular` to match our presentation. Reference: https://cybr-4580-8950.slack.com/archives/GFLQ8S1BP/p1550076849015600

File: README.md [hunk 2]
```diff
@@ -41,3 +41,38 @@ advancement of students in science, technology, engineering, and mathematics.
...
+Django is 10+ year old framework that is used by companies such as Instagram and Eventbrite. Features such as:
+
+* Django provides many templates out of the box and open source CMS possibilities to meet the requirement of a minimal degree of content management. 
+
+  * Administrators must be able to easily add new content and control its metadata. 
+  * Django templates provide uniform styling for forms and pages provided to different users. For example, a form to 'gauge the user's interest' should look and feel the same way as the form to add a new event.
+
+* Built-in user management to aid in the requirement of role-based access. 
+
+* The built-in REST framework will help the application meet the requirement of being easily accessible by external sites.
```
Comment (codykaup) [inline README.md]: The Django REST framework is a library that's not included out of the box so I'd go with something like:

> Third-party libraries like the Django REST framework...

File: README.md [hunk 3]
```diff
@@ -41,3 +41,38 @@ advancement of students in science, technology, engineering, and mathematics.
 |Development hardware| No | All | Each member will require a computer capable of running the tools needed to develop the application |
 |Cloud-based hosting environment| No | Dakota (Cody) Kaup | Hosting environment to be used for development and testing|
 |CSS framework | No | Joe | A CSS framework will be used to provide additional UI controls and site theming |
+
+## Project Metholodology
```
Comment (codykaup) [inline README.md]: I would move this entire Methodology section to between the Risk List and the Resources needed to match the milestone 1 rubric. It's a really picky thing, but it's better to be safe than sorry!
https://github.com/MLHale/CYBR4580-8950/blob/master/projects/milestone1.md

**Comment by cjkeilig** [comment]
Thanks for reviewing!

**Pull request merged**